docs/source/user-guide/troubleshooting.rst

```diff
@@ -6,17 +6,112 @@ Troubleshooting
+Solution
+--------
+
+If you are not activating your environments, start with doing that. There's more
+info at the activate-env_ page. If you are still stuck, you need to consider
+more drastic measures.
+
+  1. remove any mkl-related files from C:\Windows\System32. We recommend
+     renaming them to add .bak to the filename to effectively hide them. Observe
+     if any other software breaks. Try moving the DLL files alongside the exe of
+     the software that broke. If it works again, you can keep things in the
+     moved state - Anaconda doesn't need MKL in System32, and nothing should. If
+     you identify software that is installing software here, please contact the
+     creators of that software and inform them that their practice of installing
+     MKL to a global location is fragile and is breaking other people's software
+     and wasting a lot of time.  See the list of guilty parties below.
+  2. You may try a special DLL loading mode that Anaconda builds into Python.
+     This changes the DLL search path from system32 first to system32 last.
+     Control of this feature is done with environment variables. Only python
+     builds beyond these builds will react to these environment variables:
+
+       * python 2.7.15 build 14
+       * python 3.6.8 build 7
+       * python 3.7.2 build 8
+
+     Control environment variables:
+
+       * `CONDA_DLL_SEARCH_MODIFICATION_ENABLE`
+       * `CONDA_DLL_SEARCH_MODIFICATION_DEBUG`
+       * `CONDA_DLL_SEARCH_MODIFICATION_NEVER_ADD_WINDOWS_DIRECTORY`
+       * `CONDA_DLL_SEARCH_MODIFICATION_NEVER_ADD_CWD`
+
+List of known bad software that installs Intel libraries to C:\Windows\System32:
+
+* Amplitube, by IK Multimedia
+* ASIO4ALL, by Michael Tippach
+
+
 SSL connection errors
 =====================
 
+This is a broad umbrella of errors with many causes.  Here are some we've seen.
+
+CondaHTTPError: HTTP 000 CONNECTION FAILED
+------------------------------------------
+
+If you're on Windows and you see this error, look a little further down in the error text.  Do you see something like this?
+
+    SSLError(MaxRetryError('HTTPSConnectionPool(host=\'repo.anaconda.com\', port=443): Max retries exceeded with url: /pkgs/r/win-32/repodata.json.bz2 (Caused by SSLError("Can\'t connect to HTTPS URL because the SSL module is not available."))'))
+
+The key part there is the last bit:
+
+    Caused by SSLError("Can\'t connect to HTTPS URL because the SSL module is not available.")
+
+Conda is having problems because it can't find the openssl libraries that it needs.
+
 Cause
------
+~~~~~
+
+You may observe this error cropping up after a conda update. More recent
+versions of conda and more recent builds of python are more strict about
+requiring activation of environments. We're working on better error messages for
+them, but here's the story for now. Activation on Windows is a must. There's
+more information on activation at the activate-env_ page. When you don't activate your environment,
```

Comment:
'Activation on Windows is a must' .. well, people *can* add to PATH on Windows and it works as well as it does on other OSeS, if you exclude the problems of conda 4.6.0..4.6.4. I'd rather we removed all conda 4.6 before 4.6.6 here since that's where the bug actually was all along.

Reply:
Have we actually pinpointed the change that made 4.6.0..4.6.4 break with PATH?  I'm not sure we have.  In my view, it is the combination of the removal of Library/bin (in python 3.7) along with some change in conda.

Removing all the older 4.6 releases is pretty dramatic.  While I understand your reasons for wanting to do that, it's something we'll have to think through very carefully.

Reply:
I wrote a bigger section on this.  Please take another look.